### PR TITLE
OCT-1674: E2E: App wide fiat support - metrics view

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

**Review by aziolek — CHANGES_REQUESTED** [review]
Please add a test that should have failed and catch 3) from comment [here](https://github.com/golemfoundation/octant/pull/216#pullrequestreview-2110335910).

File: client/cypress/e2e/metrics.cy.ts [hunk 1]
```diff
@@ -2,11 +2,60 @@ import { mockCoinPricesServer, visitWithLoader } from 'cypress/utils/e2e';
...
 import {
   HAS_ONBOARDING_BEEN_CLOSED,
+  IS_CRYPTO_MAIN_VALUE_DISPLAY,
   IS_ONBOARDING_ALWAYS_VISIBLE,
   IS_ONBOARDING_DONE,
 } from 'src/constants/localStorageKeys';
 import { ROOT_ROUTES } from 'src/routes/RootRoutes/routes';
 
+const changeMainValueToFiat = () => {
+  cy.get('[data-test=Navbar__Button--Settings]').click();
```
Comment (aziolek) [inline client/cypress/e2e/metrics.cy.ts:12]: Please use `navigateWithCheck` util. This function is shared in all tests. Please move it to utils.

File: client/cypress/e2e/metrics.cy.ts [hunk 2]
```diff
@@ -2,11 +2,60 @@ import { mockCoinPricesServer, visitWithLoader } from 'cypress/utils/e2e';
...
+  cy.get('[data-test=SettingsCryptoMainValueBox__InputToggle]').uncheck();
+  cy.get('[data-test=Navbar__Button--Metrics]').click();
```
Comment (aziolek) [inline client/cypress/e2e/metrics.cy.ts:14]: Please use `navigateWithCheck` util. This function is shared in all tests. Please move it to utils.

**Review by aziolek — CHANGES_REQUESTED** [review]

File: client/cypress/e2e/metrics.cy.ts [hunk 3]
```diff
@@ -15,7 +58,7 @@ Object.values(viewports).forEach(({ device, viewportWidth, viewportHeight, isDes
        * Since Synpress needs to have valid provider to fetch the data from contracts,
        * setupMetamask is required in each test suite.
        */
-      cy.setupMetamask();
+      // cy.setupMetamask();
```
Comment (aziolek) [inline client/cypress/e2e/metrics.cy.ts:61]: Comment not required?

**Review by aziolek — CHANGES_REQUESTED** [review]

**Review by aziolek — CHANGES_REQUESTED** [review]
Where is the test added?
=> https://github.com/golemfoundation/octant/pull/246#pullrequestreview-2110390416

**Comment by jmikolajczyk** [comment]
> Where is the test added? => [#246 (review)](https://github.com/golemfoundation/octant/pull/246#pullrequestreview-2110390416)

In metrics.cy.ts file : )

**Review by aziolek — APPROVED** [review]

**Pull request merged**